yandextank/plugins/JMeter/plugin.py

```diff
@@ -5,6 +5,8 @@
 import signal
 import subprocess
 import time
+import psutil
```

Comment:
Please try avoiding this dependency. We had a lot of trouble compiling it for every system some years ago. It's like shooting sparrows from a cannon also.
Doesn't JMeter writes the port to its log?

Reply:
Ok.  And what about pid existence check ? Should i implement it also ?
Would it be correct to use /proc fs directly ?

Reply:
/proc won't work on Windows and on MacOS. JMeter's log seems like more portable solution.

Reply:
@f2nd what do you think?

Reply:
@direvius fixed. check please